packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-julia.vue

```diff
@@ -440,11 +440,50 @@ async function getAutoMapping() {
 	emit('update-state', state);
 }
 
+const initialize = async () => {
+	// Set up model config + dropdown names
+	const { modelConfiguration, modelOptions } = await setupModelInput(modelConfigId.value);
+	modelConfig.value = modelConfiguration;
+	modelStateOptions.value = modelOptions;
+
+	// Set up csv + dropdown names
+	const { filename, csv, datasetOptions } = await setupDatasetInput(datasetId.value);
+	currentDatasetFileName.value = filename;
+	csvAsset.value = csv;
+	datasetColumns.value = datasetOptions;
+
+	const node = props.node;
+	if (!node.active) return;
+
+	selectedOutputId.value = node.active;
+	selectedRunId.value = node.outputs.find((o) => o.id === selectedOutputId.value)?.value?.[0];
+
+	if (!selectedRunId.value) return;
+	lazyLoadCalibrationData(selectedRunId.value as string);
+
+	const lossVals = _.cloneDeep(node.state.intermediateLoss);
+
+	if (lossVals && lossPlot.value) {
+		const width = lossPlot.value?.offsetWidth as number;
+		renderLossGraph(lossPlot.value, lossVals, { width, height: 150 });
+	}
+
+	// Update Wizard form fields with current selected output state extras
+	extra.value = props.node.state.extra;
+};
+
+onMounted(() => {
+	initialize();
+});
+
 watch(
 	() => props.node.state.inProgressSimulationId,
 	(id) => {
 		if (id === '') {
 			unsubscribeToUpdateMessages([id], ClientEventType.SimulationSciml, messageHandler);
+			const state = _.cloneDeep(props.node.state);
+			state.intermediateLoss = lossValues;
```

Comment:
Please note this can be quite large (eg 2,000 elements of {iteration, value}). 
Is this a concern? 
I understand the state was not intended to just shove everything into do we want to save this elsewhere?